persistence/gov.go

```diff
@@ -142,7 +141,7 @@ func getParamOrFlag[T int | string | []byte](p PostgresContext, tableName, param
 		return any(v).(T), enabled, err
 
 	default:
-		log.Fatalf("unhandled type for paramValue %T", tp)
+		p.logger.Fatal().Msgf("unhandled type for paramValue %T", tp)
```

Comment:
Is there an opportunity to a `.Any()` similar to `.String()` or `.Int64()`?

Reply:
I don't think there is such method natively in `zerolog.Logger` :)